### ISSUE TITLE
tensorflow model support

### ISSUE THREAD

**Issue by Petros626** [jsk-ros-pkg/coral_usb_ros#100]
Hey,

I've read that only ssd models are supported for TFLite1 and TFLite2, so the hardware accelerator (Edge TPU) supports only ssd models until now. I saw that in the example the ssd_mobilenet_v2 i used for inference. But would the newer one (SSD MobileNet V2 FPNLite 640x640) of TensorFlow 2 also with this node?

**Comment by knorth55** [comment]
Only tensorflow1 is supported

**Comment by Petros626** [comment]
Okay thank you. So you dont know it 100%. Is there the option to ask the guy, who added the support for ssd_mobilenet_v2?

**Comment by knorth55** [comment]
Sorry, but I dont understand your question.
Mobilenet V2 is the model name, and it doesnt mean the model is for tensorflow 2

**Comment by Petros626** [comment]
Yeah thank you for the detailed explanation, if I have the time I would do so, but I'm working on my bachelor degree, so for first use it should only run on TensorFlow 1 model. 

Hahah I think the guys from TensorFlow do their best :D
